core/kazoo_services/src/kazoo_services_maintenance.erl

```diff
@@ -149,7 +149,11 @@ js_quantify_map() ->
        "        case 'user':"
        "            emit(['users', doc.priv_level], 1);"
        "            if (doc.qubicle && doc.qubicle.enabled) {"
```

Comment:
@mark2600 just checking, did this move? doc.qubicle.enabled to doc.qubicle.recipient.enabled?  If so double check the Erlang version please :)